compiler/ballerina-lang/src/test/java/org/wso2/ballerinalang/compiler/util/NumericLiteralSupportTest.java

```diff
@@ -0,0 +1,54 @@
+package org.wso2.ballerinalang.compiler.util;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class NumericLiteralSupportTest {
+
+  @Test
+  public void testStripDiscriminator() {
+    Assert.assertEquals("3", NumericLiteralSupport.stripDiscriminator("3"));
+    Assert.assertEquals("abc", NumericLiteralSupport.stripDiscriminator("abc"));
+    Assert.assertEquals("abc",
+        NumericLiteralSupport.stripDiscriminator("abcf"));
+    Assert.assertEquals("abc",
+        NumericLiteralSupport.stripDiscriminator("abcF"));
+    Assert.assertEquals("abc",
+        NumericLiteralSupport.stripDiscriminator("abcd"));
+    Assert.assertEquals("abc",
+        NumericLiteralSupport.stripDiscriminator("abcD"));
```

Comment:
I think for testng you have it the other way. So `Assert.assertEquals(actual,expected)`

Reply:
You're right. That's annoying.